### ISSUE TITLE
Bug in definition of process noise

### ISSUE THREAD

**Issue by Filippo-Tamagnini** [do-mpc/do-mpc#180]
Bug in definition of process noise

**Describe the bug**
When setting the process_noise option to True in Model.set_rhs(var_name, expr, process_noise) an error is raised. 
Internally, the Model.set_rhs method tries to append a symbolic variable to a python dictionary (self._w).
I think this bug was introduced with MX support

**Which version of do-mpc are using and how did you obtain it?**
- Develop branch
- Cloned from github

**Are you running our examples or your own code?**
nope

**Screenshots**
none

**Which system are you working on?**
 - Windows

**Additional context**
Minimal example of the bug
```
from do_mpc.model import Model

M = Model("continuous")
x = M.set_variable("_x","x")
dx = -x
M.set_rhs("x",dx,True)  #< this one will throw an error
```

**Comment by halvorak** [comment]
I agree with 52hpfans. There are also structural differences between model.set_rhs() and model.set_measurement() in the "if meas/process_noise" blocks. In set_rhs there is a check if self.symvar_type is MX/SX to set the right type for the process noise, while there is no such check for measurement noise. This is on version 4.3.3.
